src/olympia/abuse/models.py

```diff
@@ -390,7 +379,11 @@ def appeal(self, *, abuse_report, appeal_text, user, is_reporter):
             appealer=appealer_entity,
         )
         with atomic():
-            appeal_job, _ = self.__class__.objects.get_or_create(job_id=appeal_id)
+            appeal_job, _ = self.__class__.objects.get_or_create(
```

Comment:
I guess the reason why you don't separate `taget_addon` and `resolvable_in_reviewer_tools` to defaults in this query (where you do in the report method) is that an "appeal" operation is something that happens on an abuse report that has a decision, and should already have a specified target_addon/resolvable_in_reviewer_tools fields. A "report" operation is something that happens when we initially send an abuse report to cinder and so want to exclude those fields from the query, but set them if the cinder job doesn't already exists. Is that right?

Reply:
Fixed in https://github.com/mozilla/addons-server/pull/21882/commits/64eac2be7f479da0d1069810aea21b6d58256b2a